src/CachePlugin.php

```diff
@@ -45,6 +45,7 @@
      *     @var int $cache_lifetime (seconds) To support serving a previous stale response when the server answers 304
      *              we have to store the cache for a longer time than the server originally says it is valid for.
      *              We store a cache item for $cache_lifetime + max age of the response.
+     *     @var array $methods case sensitive list of request methods which can be cached.
```

Comment:
> i think its correct to allow lower case and not transform case.

But that would lead to unexpected behavior. I think we should do a case insensitive comparison or force input to be uppercase.